app.rb

```diff
@@ -56,6 +77,20 @@ def create_rental(person_id, book_title)
     end
   end
 
+  def rental_from_input
+    puts 'Enter person ID:'
+    person_id = gets.chomp.to_i
+    puts 'Enter book title:'
+    book_title = gets.chomp
+    create_rental(person_id, book_title)
+  end
+
+  def list_rentals_for_person_from_input
+    puts 'Enter person ID:'
+    person_id = gets.chomp.to_i
+    list_rentals_for_person(person_id)
+  end
+
```

Comment:
- [ Optional ] When adding a rental, the system currently requests the user to insert their personal ID. However, upon reviewing the list of all people, it appears that the personal IDs are not visible, causing confusion for the user. To address this issue and minimize confusion, I recommend making the personal IDs visible in the list of all people interface. This way, users can easily find and enter their personal ID when adding a rental, ensuring a successful rental 
creation process.
![rental](https://github.com/MarcoDDM/school_library/assets/93598189/65aaaa2b-ba8e-45c2-986b-0fba8b581f66)